Src/java/tools/cql-parsetree/src/main/java/org/cqframework/cql/tools/parsetree/Main.java

```diff
@@ -24,13 +23,13 @@ public static void main(String[] args) throws IOException {
         if (inputFile != null) {
             is = new FileInputStream(inputFile);
         }
-        CharStream input = CharStreams.fromStream(is);
+        CharStream input = CharStreams.INSTANCE.fromStream(is);
         cqlLexer lexer = new cqlLexer(input);
         CommonTokenStream tokens = new CommonTokenStream(lexer);
         tokens.fill();
         cqlParser parser = new cqlParser(tokens);
         parser.setBuildParseTree(true);
         ParserRuleContext tree = parser.library();
-        Trees.inspect(tree, parser);
+        // Trees.inspect(tree, parser);
```

Comment:
Yeah, good idea. Because of this little gui application we had inadvertently added some GUI dependencies to the compiler. That was discovered when we refactored to make it multiplatform. Another option is to simple break this out into its own "tree viewer" tool in the tools subproject, alongside the formatter.